contracts/PoolTemplate.sol

```diff
@@ -695,8 +695,12 @@ contract PoolTemplate is InsureDAOERC20, IPoolTemplate, IUniversalMarket {
 
         uint256 _debt = vault.debts(address(this));
         uint256 _totalCredit = totalCredit;
-        uint256 _deductionFromIndex = (_debt * _totalCredit * MAGIC_SCALE_1E6) /
-            totalLiquidity();
+        uint256 _totalLiquidity = totalLiquidity();
+        uint256 _deductionFromIndex = 0;
```

Comment:
I didn't know the value when totalLiquidity() return zero, so I set it to zero.

Reply:
ok!
by the way, default value of uint256 is 0, so you don't need to initialize with 0. (consume additional gas)